routers/api/v1/api.go

```diff
@@ -573,6 +573,8 @@ func RegisterRoutes(m *macaron.Macaron) {
 						Patch(reqToken(), reqRepoWriter(models.UnitTypeIssues, models.UnitTypePullRequests), bind(api.EditLabelOption{}), repo.EditLabel).
 						Delete(reqToken(), reqRepoWriter(models.UnitTypeIssues, models.UnitTypePullRequests), repo.DeleteLabel)
 				})
+				m.Post("/markdown", bind(api.MarkdownOption{}), misc.Markdown)
```

Comment:
This is an extra changes not related with the title.

Reply:
Sorry, I edited the description of the PR after the discussions above but didn't change the title. I can if you want.

This is the same as above where the change is discussed here: https://github.com/go-gitea/gitea/pull/6273/files/102bcffcb6aa60c5bee1054d5c31daa483f2c0a0#discussion_r266227788

Reply:
Ah the issue is that in order to correctly render markdown issue links you need some context. The old `/api/markdown` is insufficient - if say I want to render #6273 as a link to go-gitea/gitea#6273 I need to know that I'm rendering for go-gitea/gitea. Now we could just keep the old markdown api but we'd need to add query params etc and add the code to load them - or - we just simply add new markdown render pathways that will do that automatically for us.